spec/spec_helper.rb

```diff
@@ -53,6 +53,7 @@
 
   config.around(:each, :demo_mode_enabled) do |example|
     ENV['DEMO_MODE'] = '1'
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
Why does this belong here instead of in a global `config.before(:each)` hook?

Reply:
This is the pattern I am seeing within RSpecs, if the place we want are within the before hook, I can move it.

Reply:
It just doesn't seem like resetting the queue adapter to `:test` has any connection to setting and resetting the `DEMO_MODE` env var, but maybe I'm missing something.